docs/interchange.rst

```diff
@@ -89,16 +97,20 @@ Sequence length
 
 .. todo:: Define migration and provenance types.
 
-A tree sequence can be stored in a collection of six tables:
+A tree sequence can be stored in a collection of eight tables:
 :ref:`Node <sec_node_table_definition>`,
 :ref:`Edge <sec_edge_table_definition>`,
+:ref:`Individual <sec_individual_table_definition>`,
 :ref:`Site <sec_site_table_definition>`,
 :ref:`Mutation <sec_mutation_table_definition>`,
-:ref:`Migration <sec_migration_table_definition>`, and
+:ref:`Migration <sec_migration_table_definition>`,
+:ref:`Population <sec_population_table_definition>`, and
 :ref:`Provenance <sec_provenance_table_definition>`.
-The first two store the genealogical
-relationships that define the trees; the next two describe where mutations fall
-on those trees; the Migration table describes how lineages move across space;
+The Node and Edge tables store the genealogical
+relationships that define the trees, and the Individual table
+describes how multiple chromosomes are grouped within individuals;
+the Site and Mutation tables describe where mutations fall
+on the trees; the Migration table describes how lineages move across space;
 and the Provenance table contains information on where the data came from.
```

Comment:
insert "Only Node and Edge tables are necessary to encode the genealogical trees; Sites and Mutations are optional but necessary to encode polymorphism (sequence) data; the remainder are optional."